src/components/table/TableHeader.tsx

```diff
@@ -33,7 +34,7 @@ export default function EnhancedTableHead(props: Props): JSX.Element {
   }, [props.columns]);
 
   return (
-    <thead>
+    <TableHead>
```

Comment:
@constanzauanini do you know why we had `thead` before?
`TableHead` is required for `stickyHeader`

Reply:
I think it was for reording the table heads, does it still work?

Reply:
stickyheader works here in story example but not on web-components.. looks like we need more changes.
i will remove stickyheader changes.. lets just focus on the overriding styles in table header and cells.

Reply:
we need a max-height on table container with overflow auto.. our terraware-web views are a bit tricky for max-height at this point.

Reply:
@constanzauanini i removed sticky header fixed for now - scope of this PR is just the overriding styles